pyanaconda/ui/gui/spokes/user.py

```diff
@@ -410,33 +463,27 @@ def sensitive(self):
         # Spoke cannot be entered if a user was set in the kickstart and the user
         # policy doesn't allow changes.
         return not (self.completed and flags.automatedInstall
-                    and self.data.user.seen and not self.policy.changesok)
+                    and self.data.user.seen and not self.checker.policy.changesok)
 
     @property
     def completed(self):
         return len(self.data.user.userList) > 0
 
-    def usepassword_toggled(self, togglebutton=None, data=None):
+    def password_required_toggled(self, togglebutton=None, data=None):
         """Called by Gtk callback when the "Use password" check
         button is toggled. It will make password entries in/sensitive."""
+        password_is_required = togglebutton.get_active()
```

Comment:
The `togglebutton` variable should be required or you should have a check before calling `get_active()`  method.